src/views/RecordingsView.vue

```diff
@@ -150,19 +156,23 @@ export default {
       }
     },
     parseTags(tags) {
-      let animal = (tag) => {
-        if (!tag.animal) {
-          return "F/P";
-        } else {
-          return tag.animal;
+      if (tags.length > 0) {
+        let animal = (tag) => {
+          if (!tag.animal) {
+            return "F/P";
+          } else {
+            return tag.animal;
+          }
+        };
+        let tagString = animal(tags[0]);
+        for (let i = 1; i < tags.length; i ++) {
+          let animalName = animal(tags[i]);
+          tagString = tagString + ', ' + animalName;
         }
```

Comment:
A leaner option might be(?):

let tagString = '';
tags.forEach(tag => {
  tagString += `, ${animal(tag)}`;
});